packages/discovery-provider/scripts/lint.sh

```diff
@@ -1,4 +1,4 @@
-isort --diff . --skip ./src/tasks/core/gen --skip ./plugins
+isort . --skip ./src/tasks/core/gen --skip ./plugins
 flake8 . --exclude=./src/tasks/core/gen,./plugins
-black --diff . --exclude './src/tasks/core/gen|./plugins'
+black . --exclude './src/tasks/core/gen|./plugins'
```

Comment:
can we have some explanation for this change in the description? also, is unrelated? a reply here should be fine as well

Reply:
Yeah unrelated but basically lint fix wasnt working if you already commited the changes since it was looking at diff

Reply:
i think lint fix is still broken with this? tried it on a branch with messed up imports and it didn't remove the unused imports